webapp/niapp.yaml

```diff
@@ -0,0 +1,416 @@
+swagger: '2.0'
+info:
+  version: '1.0'
+  title: SystemLink WebApp Service
+basePath: /niapp/v1
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  ApiKeyAuth:
+    type: apiKey
+    name: x-ni-api-key
+    in: header
+security:
+  - ApiKeyAuth: []
+schemes:
+  - https
+paths:
+  '/webapps/{id}':
+    get:
+      tags:
+        - metadata
+      summary: Get webapp metadata by Id
+      description: Get webapp metadata by Id
+      operationId: get-webapp
+      parameters:
+        - $ref: '#/parameters/WebAppId'
+      responses:
+        200:
+          $ref: '#/responses/GetWebAppResponse'
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          $ref: '#/responses/NotFound'
+    put:
+      tags:
+        - metadata
+      summary: Update existing webapp metadata
+      description: Update webapp metadata by Id
+      operationId: update-webapp
+      parameters:
+        - $ref: '#/parameters/WebAppId'
+        - $ref: '#/parameters/UpdateWebAppRequest'
+      responses:
+        200:
+          $ref: '#/responses/UpdateWebAppResponse'
+        400:
+          $ref: '#/responses/ValidationError'
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          $ref: '#/responses/NotFound'
+    delete:
+      tags:
+        - metadata
+      summary: ''
+      description: Delete webapp by id
+      operationId: delete-webapp
+      parameters:
+        - $ref: '#/parameters/WebAppId'
+      responses:
+        200:
```

Comment:
Generally the success response to a DELETE should be a 204 if the response is empty, and a 200 if the response contains other info.

Reply:
This is the documentation of the existing implementation.